apps/api/e2e/tests/price-fetching.test.ts

```diff
@@ -32,10 +31,9 @@ const BASE_TOKENS = {
 };
 
 describe("Price Fetching", () => {
-  // Create variables for authenticated clients
-  let adminClient: ApiClient;
-  let client: ApiClient;
+  // Create variables for api keys
   let adminApiKey: string;
+  let clientApiKey: string;
```

Comment:
This is the test file I updated to use the SDK.  Maybe spend a bit more time looking here to see how the SDK works.